src/testrun.rs

```diff
@@ -46,63 +48,145 @@ impl Display for Outcome {
     }
 }
 
-#[derive(Clone, Debug)]
+static FRAMEWORKS: LazyLock<Vec<(&str, Framework)>> = LazyLock::new(|| {
+    vec![
+        ("pytest", Framework::Pytest),
+        ("vitest", Framework::Vitest),
+        ("jest", Framework::Jest),
+        ("phpunit", Framework::PHPUnit),
+    ]
+});
+static EXTENSIONS: LazyLock<Vec<(&str, Framework)>> =
+    LazyLock::new(|| vec![(".py", Framework::Pytest), (".php", Framework::PHPUnit)]);
```

Comment:
```suggestion
static FRAMEWORKS: &[(&str, Framework)] = &[
        ("pytest", Framework::Pytest),
        ("vitest", Framework::Vitest),
        ("jest", Framework::Jest),
        ("phpunit", Framework::PHPUnit),
    ];
static EXTENSIONS: &[&str, Framework)] = &[(".py", Framework::Pytest), (".php", Framework::PHPUnit)];
```

You can just create a fully static slice here, instead of lazily allocating a vec on the heap on first access.